daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -49,13 +49,22 @@ main = do
         mvnPath <- locateRunfiles "mvn_dev_env/bin"
         tarPath <- locateRunfiles "tar_dev_env/bin"
         yarnPath <- takeDirectory <$> locateRunfiles (mainWorkspace </> yarn)
+        -- NOTE(Sofia): We don't use `script` on Windows.
```

Comment:
Might be nice to add a link to the ammonite issue explaining why we use `script` at all.

Reply:
Added a comment & link below, where we call script